module-apps/apps-common/widgets/TimeMinuteSecondWidget.cpp

```diff
@@ -100,9 +115,12 @@ namespace gui
 
     void TimeMinuteSecondWidget::updateTime(std::uint32_t seconds)
     {
+        if (!totalSeconds) {
```

Comment:
I'd explicitly compare it against zero so that it's clear that it's a numeric value, not a boolean type, pointer, `std::optional` etc.

Reply:
done